CMakeLists.txt

```diff
@@ -108,10 +108,11 @@ link_directories(${CCD_LIBRARY_DIRS})
 option(FCL_WITH_OCTOMAP "octomap library support" ON)
 set(FCL_HAVE_OCTOMAP 0)
 if(FCL_WITH_OCTOMAP)
-  if(PKG_CONFIG_FOUND)
+  find_package(octomap QUIET)
+  if(NOT OCTOMAP_INCLUDE_DIRS AND NOT OCTOMAP_LIBRARY_DIRS AND PKG_CONFIG_FOUND)
```

Comment:
Wouldn't it be simpler to use `octomap_FOUND` here? Maybe you used `OCTOMAP_INCLUDE_DIRS AND` and `OCTOMAP_LIBRARY_DIRS` because of the case inconsistency? If so, it would be good to leave a short note here why we don't use `XXX_FOUND` for maintenance in the future. I think the case inconsistency should be fixed in the upstream, though.